python/tk_syntheyes/inbuilt_apps/playblast.py

```diff
@@ -0,0 +1,272 @@
+import os
+import time
+from threading import Thread
+
+import sgtk
+import SyPy3
+
+from engine import SynthEyesEngine
+from tk_syntheyes.inbuilt_app import InbuiltApp
+
+from PySide2.QtCore import *
+from PySide2.QtGui import *
+from PySide2.QtWidgets import *
```

Comment:
Bitte niemals wildcard imports benutzen. (Ich weiß das wird bei QT / Pyside gerne mal gemacht. Aber das ist generell bad-practice.

Auszug aus pep8: 

`Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn’t known in advance).`